metadata-models/src/main/pegasus/com/linkedin/ml/metadata/MLFeatureProperties.pdl

```diff
@@ -8,6 +9,11 @@ import com.linkedin.common.VersionTag
  */
 record MLFeatureProperties {
 
+  /**
+   * Name of the MLFeature
+   */
+  name: optional string
```

Comment:
Is this not encoded in the URN? 
Also need to add new elements to the end of the record for backward compatibility I believe.